base_mixin_restrict_field_access/models/restrict_field_access_mixin.py

```diff
@@ -229,3 +284,17 @@ def _restrict_field_access_is_field_accessible(self, field_name,
         whitelist = self._restrict_field_access_get_field_whitelist(
             action=action)
         return field_name in whitelist
+
+
+class RestrictedExport(Export):
```

Comment:
move this class to `controllers/main.py`, that's the conventional place where controllers should live